samples/system-test/automlVision.test.js

```diff
@@ -0,0 +1,128 @@
+
+const testDataSetName = `testDataSet`;
+const dummyDataSet = `dummyDataSet`;
+const testModelName = "dummyModel";
+const sampleImage = `./automl/resources/testImage.jpg`;
+const sampleImage2 = `./automl/resources/testImage2.jpg`;
+
+// skipped because it's been taking too long to delete datasets
+test.skip(`It should create a create, list, and delete a dataset`, async t => {
+  // Check to see that this dataset does not yet exist
+  let output = await tools.runAsync(`${cmdDataset} listDatasets`);
+  t.false(output.includes(testDataSetName));
+
+  // Create dataset 
+  output = await tools.runAsync(`${cmdDataset} createDataset -n "${testDataSetName}"`);
+  const dataSetId = output.split(`\n`)[1].split(`:`)[1].trim();
+  t.true(output.includes(`Dataset display name:  ${testDataSetName}`));
+
+  // delete dataset 
+  output = await tools.runAsync(`${cmdDataset} deleteDataset -i "${dataSetId}"`);
+  t.true(output.includes(`Dataset deleted.`));
+});
+
+// see : https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/vision/automl/model_test.py
+// we make two models running this test, see hard-coded workaround below
+test.skip(`It should create a dataset, import data, and start making a model`, async t => {
+    // Check to see that this dataset does not yet exist
+    let output = await tools.runAsync(`${cmdDataset} listDatasets`);
+    t.false(output.includes(dummyDataSet));
+  
+    // Create dataset 
+    output = await tools.runAsync(`${cmdDataset} createDataset -n "${dummyDataSet}"`);
+    const dataSetId = output.split(`\n`)[1].split(`:`)[1].trim();
+    t.true(output.includes(`Dataset display name:  ${dummyDataSet}`));
+
+  // Import Data
+  output = await tools.runAsync(`${cmdDataset} importData -i "${dataSetId}" -p "gs://nodejs-docs-samples-vcm/flowerTraindata20lines.csv"`);
+  t.true(output.includes(`Data imported.`));
+
+  // Check to make sure model doesn't already exist
+  output = await tools.runAsync(`${cmdModel} listModels`);
+  t.false(output.includes(`${testModelName}`));
+
+  // begin training dataset, getting operation ID for next operation 
+  output = await tools.runAsync(`${cmdModel} createModel -i "${dataSetId}" -m "${testModelName}" -t "2"`);
+  const operationName = output.split(`\n`)[0].split(`:`)[1].trim();
+  t.true(output.includes(`Training started...`));
+
+  // poll operation status, here confirming that operation is not complete yet
+  output = await tools.runAsync(`${cmdModel} getOperationStatus -i "${dataSetId}" -o "${operationName}"`);
+  t.true(output.includes(`done: false`));
+});
+
+test(`It should display evaluation from prexisting model`, async t => {
+  const flowersModelId = `ICN723541179344731436`;
+  const flowersDatasetId = `ICN7628761482635807401`;
+  const flowersname = `projects/203278707824/locations/us-central1/datasets/ICN7628761482635807401`;
+  const flowersDisplayName = `flowersTest`;
+  const donotdeleteModelId = `ICN723541179344731436`;
+  
+  //confirm dataset exists
```

Comment:
Nit: add space, and capitalize comments. (Apply this change throughout this PR.)